teku/src/main/java/tech/pegasys/teku/cli/deposit/RegisterAction.java

```diff
@@ -114,7 +114,7 @@ public void displayConfirmation(final int totalNumberOfDeposits) {
       return;
     }
     System.out.println("Transaction cancelled.");
-    shutdownFunction.accept(0);
+    shutdownFunction.accept(1);
```

Comment:
is this the only case we needed to change? I know this was the concrete example given in the ticket, but if the transaction fails to submit somehow, like the eth1 node didn't respond or something i guess? Insufficient credit to consume maybe?

Reply:
That case is already covered here in `DepositRegisterCommand.java`: 
```
  @Override
  public void run() {
    final BLSKeyPair validatorKey = getValidatorKey();

    try (final RegisterAction registerAction =
        registerParams.createRegisterAction(verboseOutputParam.isVerboseOutputEnabled())) {
      registerAction.displayConfirmation(1);
      registerAction.sendDeposit(validatorKey).get();
    } catch (final Throwable t) {
      SUB_COMMAND_LOG.sendDepositFailure(t);
      shutdownFunction.accept(1);
    }
    shutdownFunction.accept(0);
  }
```